Packs/EDL/Integrations/EDL/README.md

```diff
@@ -27,36 +27,44 @@ Unlike `PAN-OS EDL Management`, this integration hosts the EDL on the Cortex XSO
+| Append string to list              | String to add to end of published list. Supports newline characters (\n).                                                                                                                                                                            | False        |
+| IP Collapsing                      | For use with PAN-OS (text) format - collapse method for IPs (none, range, CIDR).                                                                                                                                                                     | False        |
+| PAN-OS: drop invalid URL entries   | For use with PAN-OS (text) format - if selected, any URL entry that is not compliant with PAN-OS URL format is dropped instead of being rewritten.                                                                                                   | False        |
+| McAfee Gateway: Indicator List Type | For use with McAfee Web Gateway format - set the indicator list type.                                                                                                                                                                                | False        |
+| Symantec ProxySG: Default Category | For use with Symantec ProxySG format - set the default category for the output.                                                                                                                                                                      | False        |
+| Symantec ProxySG: Listed Categories | For use with Symantec ProxySG format - set the categories that should be listed in the output. If not set, will list all existing categories.                                                                                                        | False        |
+| Show CSV formats as Text           | If selected, CSV format will appear in a textual webpage instead of initiating a file download.                                                                                                                                                      | False        |
+| XSOAR Indicator Page Size          | Internal page size used when querying Cortex XSOAR for the indicators.                                                                                                                                                                               | False        |
+| Maximum CIDR prefix size   | CIDRs with prefix size smaller than this value will not be included. Default is 8.                                                                                                                                                                   | True         |
```

Comment:
CIDR network prefix bits size | CIDRs with a smaller network prefix bits number will not be included. For example - if the number is 8, then 0.0.0.0/2 will be excluded from the list. | False